cluster/images/kubelet/Makefile

```diff
@@ -1,21 +1,21 @@
-# build the kubelet image.
+# Build the kubelet image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-# TODO: figure out the best base image
-# TODO: figure out how to best source the nsenter binary, if necessary
-# TODO: figure out how to make a single source of version info for these
-#       images
-VERSION=v0.16
```

Comment:
Yes, but if we leave a default value here user might not even realize that we are setting some defaults. Any value that we put here will out-date pretty fast. I think it's cleaner to force user to pass it.